src/app/features/management/user-management-view/edit-user-view/edit-user-view.component.ts

```diff
@@ -0,0 +1,105 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Loader } from '../../../../core/util/loader';
+import { NotificationService } from '../../../../core/services/notification.service';
+import { FormBuilder, Validators } from '@angular/forms';
+import { MDSError, MDSErrorCode } from '../../../../core/util/errors';
+import { Subscription } from 'rxjs';
+import { ActivatedRoute, Router } from '@angular/router';
+import { UserService } from '../../../../core/services/user.service';
+
+/**
+ *  View for updating a user.
+ */
+@Component({
+  selector: 'app-edit-user-view',
+  templateUrl: './edit-user-view.component.html',
+  styleUrls: ['./edit-user-view.component.scss'],
+})
+export class EditUserView implements OnInit, OnDestroy {
+
+  /**
+   * Loader for when updating a user and awaiting the response.
+   */
+  updatingUser = new Loader();
+
+  form = this.fb.nonNullable.group({
+    username: this.fb.nonNullable.control<string>('',[Validators.required]),
+    firstName: this.fb.nonNullable.control<string>('',[Validators.required]),
+    lastName: this.fb.nonNullable.control<string>('',[Validators.required]),
+    isAdmin: this.fb.nonNullable.control<boolean>(false),
+    isActive: this.fb.nonNullable.control<boolean>(true),
+  });
+
+  private s: Subscription[] = [];
+  id = '';
+
+  constructor(private userService: UserService, private notificationService: NotificationService, private fb: FormBuilder,
+              private router: Router, private route: ActivatedRoute) {
+  }
+
+  ngOnDestroy(): void {
+    this.s.forEach(s => s.unsubscribe());
+  }
+
+  ngOnInit(): void {
+    // Get id of current user from router path.
+    this.s.push(this.route.params.subscribe(params => {
+      this.id = params['id'];
+      this.userService.getUserById(this.id).subscribe(u => {
+        this.form.patchValue({
+          username: u.username,
+          firstName: u.firstName,
+          lastName: u.lastName,
+          isAdmin: u.isAdmin,
+          isActive: u.isActive,
+        });
+      });
```

Comment:
missing call to `updatingUser`.
```suggestion
this.updatingUser.load(this.userService.getUserById(this.id)).subscribe(u => {
        this.form.patchValue({
          username: u.username,
          firstName: u.firstName,
          lastName: u.lastName,
          isAdmin: u.isAdmin,
          isActive: u.isActive,
        });
      });
```